lib/gui/modules/drive-scanner.js

```diff
@@ -21,13 +21,60 @@ const _ = require('lodash')
 const EventEmitter = require('events').EventEmitter
 const drivelist = require('drivelist')
 const settings = require('../models/settings')
+const usbboot = require('../../shared/sdk/usbboot')
 
 const DRIVE_SCANNER_INTERVAL_MS = 2000
 const DRIVE_SCANNER_FIRST_SCAN_DELAY_MS = 0
 const emitter = new EventEmitter()
 
 /* eslint-disable lodash/prefer-lodash-method */
 
+/**
+ * @summary
+ *
+ * @param {Function} callback - two argument function callback of (err, result)
+ *
+ * @example
+ */
+const lister = (callback) => {
+  drivelist.list((driveError, drives) => {
+    if (driveError) {
+      return callback(driveError)
+    }
+
+    const typeDrives = _.map(drives, (drive) => {
+      return _.assign({}, drive, {
+        type: 'BLOCK_DEVICE'
+      })
+    })
+
+    return usbboot.scan().then((usbDevices) => {
+      const typeUsbDevices = _.map(usbDevices, (usbDevice) => {
+        const description = `Bus ${usbDevice.busNumber} / Address ${usbDevice.deviceAddress}`
```

Comment:
If such a field is needed, it'd probably be better if it was added into the `usbDevice` object in @jviotti 's usbboot PR? Much like the `usbDevice.name` field you're also making use of below.